editor/vim/coc-settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "languageserver": {
+        "jsonnet": {
+        "command": "jsonnet-language-server",
+        "args": ["-t"],
+        "rootPatterns": [".git/"],
```

Comment:
I would suggest to also add `jsonnetfile.json` as root directory indicator.

Reply:
+1 doing